src/components/NewLexemeForm.vue

```diff
@@ -38,6 +38,16 @@ const lexicalCategory = computed( {
 		store.commit( SET_LEXICAL_CATEGORY, newLexicalCategory );
 	},
 } );
+
+const submitMsg = $messages.get( 'wikibaselexeme-newlexeme-submit' );
+const termsOfUseURL = $messages.get( 'copyrightpage' );
```

Comment:
I suggest renaming this (e.g. to `termsOfUseURL`), since it won’t actually be a URL anymore.